app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/CustomOptions.php

```diff
@@ -603,7 +604,8 @@ protected function getStaticTypeContainerConfig($sortOrder)
                         'component' => 'Magento_Ui/js/form/components/group',
                         'breakLine' => false,
                         'showLabel' => false,
-                        'additionalClasses' => 'admin__field-group-columns admin__control-group-equal',
+                        'additionalClasses' => 'admin__field-group-columns admin__control-group-equal '
+                            . 'admin__control-grouped',
```

Comment:
Additional classes can be array, you don't need to concatenate strings

Reply:
fixed